Index: Sofware_Project_20250517/Agricultur_Sector/secondaddedfileinto main2
===================================================================
diff --git a/Sofware_Project_20250517/Agricultur_Sector/secondaddedfileinto main2 b/Sofware_Project_20250517/Agricultur_Sector/secondaddedfileinto main2
--- a/Sofware_Project_20250517/Agricultur_Sector/secondaddedfileinto main2	
+++ b/Sofware_Project_20250517/Agricultur_Sector/secondaddedfileinto main2	
@@ -1,0 +1,1 @@
+hi from text main2
\ No newline at end of file
